### DISCUSSION TITLE
No clickable tx links

### DISCUSSION THREAD

**Discussion by dbwebapp** [BitCannaGlobal/WeedWallet#90] [General]
Thos following links are usually cllilckable so you can be redirected to the scan explorer and see the transaction in details. Now they are not clickable.

![Screenshot 2023-05-25 at 22 06 50](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/0f6935bb-6e1b-43a3-926c-ccbad46c35a5)

**Comment by TJ-BitCanna** [comment]
Thanks for notifying. We'll look into this.

**Comment by TJ-BitCanna** [comment]
Suggested issue:
- make the block number and block header clickable directing to an explorer

**Comment by TJ-BitCanna** [comment]
- Link added in the transactions history
- Clickable links added in the undelegation and redelegations blocks on the validator specific page
